src/js/common/schemaform/widgets/TextWidget.jsx

```diff
@@ -12,7 +12,7 @@ export default function TextWidget(props) {
         maxLength={props.schema.maxLength}
         autoComplete={props.options.autocomplete || false}
         className={props.options.widgetClassNames}
-        value={props.value || ''}
```

Comment:
Right, leaving the empty string in there would be fine, but a bit redundant is all. Either the input's `value` is whatever falsey value `props.value` was, or the specific falsey value of `''` before turning into `undefined`. I just figured having that `|| ''` looked an awful lot like we wanted an empty string, when what we really want is `undefined` in the end.